src/test/ui/parser/issues/issue-94340.stderr

```diff
@@ -0,0 +1,28 @@
+error: an inner attribute is not permitted in this context
+  --> $DIR/auxiliary/issue-94340-inc.rs:2:1
+   |
+LL | #![deny(rust_2018_idioms)]
+   | ^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+   = note: inner attributes, like `#![no_std]`, annotate the item enclosing them, and are usually found at the beginning of source files
+   = note: outer attributes, like `#[test]`, annotate the item following them
+
+error: an inner attribute is not permitted in this context
+  --> $DIR/auxiliary/issue-94340-inc.rs:3:1
+   |
+LL | #![deny(unused_must_use)]
+   | ^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+   = note: inner attributes, like `#![no_std]`, annotate the item enclosing them, and are usually found at the beginning of source files
+   = note: outer attributes, like `#[test]`, annotate the item following them
+
+error: expected item after attributes
```

Comment:
I thought about this error again. I _think_ it shouldn't be emitted (just like I originally wrote in the issue before I struck it through).
My reasoning is: You are allowed to `include!` an empty file (basically the `<eof>`, end of file, token) and by extension, we should not emit extra errors (like this one) when we include `<invalid_inner_attr> <invalid_inner_attr> <eof>`. We are still emitting the `is not permitted in this context` error, so that's fine.

Reply:
Yeah, I agree. We should completely ignore the attribute. It might cause some knock down errors either way (from code that uses the expected expanded code), but at least it won't be outright wrong.